### ISSUE TITLE
UEFI Firmware Settings entry

### ISSUE THREAD

**Issue by gilbertohasnofb** [EvanPurkhiser/rEFInd-minimal#64]
Hi, I just migrated from Grub to rEFInd, which I have been using with your theme (and which is absolutely gorgeous, by the way). 

The default rEFInd menu had a lot of other icons for settings which I understand your minimal theme removed. I was wondering if it would be possible for me to have an entry for the 'UEFI Firmware Settings'. Back with Grub, I had such entry which executed `fwsetup`. This booted the computer to the UEFI screen showing me what function keys I could press to go into BIOS, boot from USB sticks, etc. Is it possible to manually add such entry to rEFInd using your theme?

**Comment by gilbertohasnofb** [comment]
According to https://docs.slackware.com/howtos:slackware_admin:uefi_triple_boot_with_refind_on_slackware#non-bootloader_tools, the default theme has a command called `showtools` which configures what tools are shown at the bottom. Setting it to `firmware` will add the entry to UEFI Firmware Settings (e.g. setting it as `showtools firmware,shell,shutdown`). Is there some equivalent command with your theme?

**Comment by EvanPurkhiser** [comment]
I believe showtools should work just fine, but it may not have icons that match the theme.

This theme doesn't actually add any new commands, so everything you see in the [refind documentation](https://www.rodsbooks.com/refind/configfile.html) you may specify in the theme file.

**Comment by gilbertohasnofb** [comment]
> I believe showtools should work just fine, but it may not have icons that match the theme.

Ok, I figured out what was going on. I was setting up `showtools` in `refind.conf` but you are overwriting it in `theme.conf`. All I needed was to edit it in `theme.conf` and set it to `showtools shutdown, firmware`. I also made a copy of the file `func_reset.png` and renamed it `func_firmware.png`, so now I have an arrow down for shutdown and an arrow up for the firmware menu with BIOS and advanced boot options. It's all looking really good now, thanks a bunch!
